plugins/org.jboss.tools.openshift.common.ui/plugin.xml

```diff
@@ -258,8 +258,7 @@
 						<iterate
 							ifEmpty="false"
 							operator="and">
-							<instanceof
-								value="org.jboss.tools.openshift.common.core.connection.IConnection" />
+							<adapt	type="org.jboss.tools.openshift.common.core.connection.IConnection" />
```

Comment:
For my own edification, does this mean the class has to impl adaptable so that it can return an IConnection? Or does the platform provide some magic that determines if it can be cast to that interface?